organisms/organisation-unit-tree/new-features/recursive-selection.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Recursive selection of org units
+
+  Background:
+    Given an org unit tree with recursive selection mode enabled
+
+  Scenario:
+    Given an org unit tree with a root unit with descendants
```

Comment:
Sentences should be valid grammatically, this one lacks a verb. Maybe `Given an org unit tree has a root and descendants nodes`? (Same for the `Background` sentence)

Reply:
👍 Fixed.